azure-pipelines/main-pipeline.yml

```diff
@@ -62,6 +63,29 @@ stages:
     
     steps:
 
+    
+    # Static Analysis with SonarCloud. Pointing to Java Binaries and exluding the JDK
+    # Tar file that will be installed in the pipeline. This step scans all services.
+
+    - task: SonarCloudPrepare@1
+      inputs:
+        SonarCloud: 'SonarCloud'
+        organization: '$(sonarOrg)'
+        scannerMode: 'CLI'
+        configMode: 'manual'
+        cliProjectKey: '$(sonarProjectKey)'
+        cliProjectName: '$(sonarProjectName)'
+        cliSources: '.'
+        extraProperties: |
+            sonar.java.binaries=**/src
+            sonar.exclusions=./jdk/**/*
```

Comment:
keep indentation consistent

Reply:
I'm still sort of unsure what's wrong with the indentation here. But I refactored the whole .yml file so it may have been fixed.

Reply:
the properties value was indented by 4 extra spaces instead of the 2 used elsewhere. fixed during the refactor.